src/toCurrency/toCurrency.ts

```diff
@@ -20,20 +24,22 @@ export function toCurrency(value: number | string, options: ToCurrencyOptions =
             num = val;
         } else {
             num = Math.floor(val);
-            cents = options && options.showDecimals ? null : Number((val - Math.floor(val)).toFixed(options && options.numOfDecimals ? options.numOfDecimals : 2).replace("0.", ""));
+            if (options?.showDecimals !== false) {
```

Comment:
why can't we just use `if (options?.showDecimals)`?

Reply:
Because the default is true, so `undefined` is considered OK. Only when you manually set it to `false` then I will not show the decimal places